src/Microsoft.DocAsCode.Build.SchemaDriven/Processors/MarkdownInterpreter.cs

```diff
@@ -35,7 +35,7 @@ private static string MarkupCore(string content, IProcessContext context, string
         {
             var host = context.Host;
 
-            var mr = host.Markup(content, context.GetOriginalContentFile(path));
+            var mr = host.Markup(content, context.GetOriginalContentFile(path), false, true);
```

Comment:
this actually enables validation - is this your purpose for this PR?

Reply:
Yes, this PR is targeting to enable validation and add YamlMime to the document, then validation library could test it. 
Btw, I understand the next build is scheduled to Nov. 18, I think I get plenty of time for testing before going to production.

Reply:
I have a offline talk with Renze, I will complete this PR after validation library is ready

Reply:
Update title to reflect the real purpose of this pr.